lib/ftu/filters.dart

```diff
@@ -1,25 +1,145 @@
+
+import'../ftu/parsing.dart';
+import 'dart:async';
+import 'dart:convert';
 import 'package:flutter/material.dart';
```

Comment:
To follow effective_dart styling, move the `package` imports to the top and in alphabetical order.

Reply:
https://dart.dev/guides/language/effective-dart/style